public/kibi_timeline.js

```diff
@@ -238,6 +230,12 @@ define(function (require) {
                   }
                 }
                 events.push(e);
+              } else {
+                if ($scope.notifyNull) {
```

Comment:
ok better now :-)
Cold you please change the name of the flag to notifyDataErrors as it should trigger if the value is either 
undefined null or empty string

Make sure that if the value is an empty string this is also triggered